docs/aws/audit/elbmonitoring/rules/classic_elb_in_use_remediation.mdx

```diff
@@ -1,6 +1,217 @@
+            elb_client.delete_load_balancer(LoadBalancerName=elb_name)
+    else:
+        print("No Classic ELBs to delete.")
+
+# Call the function to delete Classic ELBs
+delete_classic_elbs()
+```
+
+### Summary
+1. **Install Boto3**: Ensure Boto3 is installed.
+2. **Initialize Boto3 Client**: Set up the ELB and ELBv2 clients.
+3. **List and Check Classic ELBs**: List all Classic ELBs and check if any are in use.
+4. **Automate the Prevention**: Create a script to delete Classic ELBs if found.
+
+By following these steps, you can prevent the use of Classic ELBs in your AWS environment using Python scripts.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the Amazon EC2 console at https://console.aws.amazon.com/ec2/.
+
+2. In the navigation pane, under LOAD BALANCING, choose Load Balancers.
+
+3. In the Load Balancer type column, you can see the type of each load balancer. If any of them are listed as "Classic Load Balancer", it indicates that a Classic ELB is in use.
+
+4. For a more detailed view, select the Classic Load Balancer. In the Description tab below, you can see more details about the selected load balancer.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local system. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your AWS Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all Classic Load Balancers: Run the following command to list all the Classic Load Balancers in your AWS account.
+
+   ```
+   aws elb describe-load-balancers
+   ```
+
+   This command will return a JSON output with the details of all the Classic Load Balancers.
+
+3. Parse the output: You can parse the output using a tool like jq to get the names of the Classic Load Balancers. Here is an example command:
+
+   ```
+   aws elb describe-load-balancers | jq -r '.LoadBalancerDescriptions[].LoadBalancerName'
+   ```
+
+4. Check for Classic Load Balancers: If the above command returns any names, it means that there are Classic Load Balancers in use. If it doesn't return any names, it means that there are no Classic Load Balancers in use.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>